tests/queryset/modify.py

```diff
@@ -94,6 +96,30 @@ def test_modify_with_fields(self):
         self.assertEqual(old_doc.to_mongo(), {"_id": 1})
         self.assertDbEqual([{"_id": 0, "value": 0}, {"_id": 1, "value": -1}])
 
+    @needs_mongodb_v26
+    def test_modify_with_push(self):
```

Comment:
I'd just assert `blog.tags` is correct without doing the ugly `to_mongo` calls and comparing the entire docs.